manager/src/test/java/io/gingersnapproject/hotrod/HotRodOperationsTest.java

```diff
@@ -48,15 +48,12 @@ public void afterEach() {
 
    @Test
    public void testGetAll() {
-      Map<String, String> values = new HashMap<>();
-      values.put("key1", "value1");
-      values.put("key3", "value3");
-      values.put("key2", "value3");
-
-      cache.put("key1", "value1");
-      cache.put("key3", "value3");
-      cache.put("key2", "value3");
-
+      var values = Map.of(
+              "key1", "{\"fullname\": \"value1\"}",
+              "key2", "{\"fullname\": \"value2\"}",
+              "key3", "{\"fullname\": \"value3\"}"
```

Comment:
Update the values to JSON as `Caches#denormalize` now fails if the value is not JSON. IMO this is acceptable as we only expect a JSON to be put from the db-syncer and we always create a JSON object when reading from the DB directly.